.github/workflows/rust.yml

```diff
@@ -44,3 +44,9 @@ jobs:
       run: cargo test
     - name: Run clap & curl tests
       run: cargo test --features http-curl,clap-parse
+
+    - uses: baptiste0928/cargo-install@v2
+      with:
+        crate: cargo-msrv
```

Comment:
`cargo-msrv` is failing to install on windows (looks like [an issue with the version of tokio](https://stackoverflow.com/questions/76173195/my-rust-project-is-broken-all-of-the-sudden)). If you know how to get cargo to install the cargo-msrv with the right tokio & ntapi versions that would be ideal, if not just move this to the `doc` job so it only runs on ubuntu.

Reply:
Personally, I verify MSRV by running regular cargo commands.  I then have [renovatebot](https://github.com/clap-rs/clap/blob/master/.github/renovate.json5#L8) update MSRV for me

Reply:
Yeah, cargo-msrv is not really required here.

Feel free to remove.